nkpremices/Banka
# #165156168 Server setup (server side framework)

#### What does this PR do?
Have the server side configured 

#### Description of Task to be completed?
* Create a branch called back-end which will be parent branch of all back end feature branches
* Setup node js
* Setup express Js
* Setup body parser
* Setup babel 
* Setup Eslint with airBnb
* Rearrange previous code accordingly 
* Write the welcome message

#### How should this be manually tested?
After cloning to the repo:
* Run the command npm install to install all the necessary packages
* Then start the server with the command npm start
* with postman, test the endpoint GET / to see the welcome message
